my-site.rb

```diff
@@ -0,0 +1,50 @@
+require "sinatra"
+require "./lib/scrabble_master.rb"
+
+class MySite < Sinatra::Base
+	# Creates the home page which links to a scoring page
+	get "/" do
+		@title = "Scrabble"
+		erb :index
+	end
+
+	get "/index.html" do 
+		@title = "Scrabble"
+		erb :index
+	end
+
+	# Creates the score page from the home page
+	get "/score.html" do 
+		@title = "Scrabble Scorecard"
+		erb :score
+	end
+
+	# Writing out the score using POST
+	post "/score.html" do
+		@word_scored = true
```

Comment:
I think this boolean variable that signifies whether the word has been scored or not can be removed, and your view code could check for the presence of the `@score` variable. If `@score` is not nil in the `score.erb` template, then this must have been a POST request and our Sinatra code figured out the score and set the variable.